### PR TITLE
EDU-1309 Move role info into modal dialog

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1309

### CODE REVIEW

File: src/components/info.js [hunk 1]
```diff
@@ -4,24 +4,24 @@ import PropTypes from 'prop-types';
...
-                'Info-icon',
-                'Info-icon--help',
-                { 'Info-icon--afterContent': iconAfterContent }
-              )
-            }
-            />
-        </Tooltip>
-      )}
-      {!tooltip && <InfoCircleOutlined className="Info-icon" />}
+      {tooltip ? <Tooltip title={tooltip}>{icon}</Tooltip> : icon}
```
Comment (adrianaluchian) [inline src/components/info.js:24]: Honestly I would rather not repurpose this component and use the questionmark icon from editing sections instead (even if it's not as nice of an icon).
I think we should keep constant with patterns patterns we use and so far this info is hover only, while that section icon opens a dialog. And the fact that the helptext has to be "Click here [...]" is an indicator that the user will expect something else.

Comment (ahelmberger) [reply src/components/info.js]: Ok, makes sense. I reverted the `info` component. But I used the AntD question mark ...

localhost_3000_admin_tab=user-accounts

..., as the custom one hurts the eye so much:

localhost_3000_admin_tab=user-accounts (1)

Comment (adrianaluchian) [reply src/components/info.js]: Yeah, I know 😓 That one is by far not the best icon we've got, from the custom ones.
The AntD one works well 👍 
As tooltip, maybe we can now simply use the same one as for the document section help, `common.openHelp`, what do you think? Short and to the point, more chances of the text actually being read by the user (and consistent).

Comment (ahelmberger) [reply src/components/info.js]: Definitely :+1: Done!

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**